.gitignore

```diff
@@ -4,3 +4,4 @@ vendor
 tests/SimpleThings/Tests/EntityAudit/cache
 tests/SimpleThings/Tests/EntityAudit/Proxies
 .php_cs.cache
+.idea
```

Comment:
please revert - this is not a project specific ignore pattern and should be maintained in your global gitignore (system-, ide-dependent ignores)